sig-advocacy-and-outreach/gsoc/README.md

```diff
@@ -6,5 +6,5 @@ KubeSphere in [Google Summber of Code (aka GSoC)](https://summerofcode.withgoogl
 | --- | --- | --- |
 | [KubeSphere Jenkins Client Refactor](kubeSphere-jenkins-client-refactor.md) <br/>Split the Jenkins client from KubeSphere. Or use an existing Jenkins client. </br>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, REST API/OpenAPI, Jenkins, Kubernetes |
 | [KubeSphere Tekton Integration](kubeSphere-tekton-integration.md) <br/>Integrate Tekton as an alternative CI/CD engine of KubeSphere DevOps. <br/>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, Tekton, Jenkins, Kubernetes |
-| [Customize receiving users via webhook](customize-receiving-users-via-webhook.md) <br/>Make the notification manager support to get a user list that will receive the notifications from a webhook. <br/> Potential Mentor(s): [benjaminhuo](https://github.com/benjaminhuo), [lei](https://github.com/wanjunlei)| Notification Manager | Golang, REST API, Kubernetes |
+| [Customize Notification Receivers Using a Webhook](customize-receiving-users-via-webhook.md) <br/>Enable [Notification Manager](https://github.com/kubesphere/notification-manager) to obtain a notification receiver list from a webhook so that users can customize the notification receivers. <br/>Potential Mentor(s): [Benjamin Huo](https://github.com/benjaminhuo), [Wanjun Lei](https://github.com/wanjunlei) | Notification Manager | Golang, REST API, Kubernetes |
```

Comment:
As I mentioned before. You need to update the link here. Let's say, please change `[Customize Notification Receivers Using a Webhook](customize-receiving-users-via-webhook.md)` to `[Customize Notification Receivers Using a Webhook](customize-notification-receivers-using-a-webhook.md)`